app/src/main/java/com/hedvig/app/feature/insurance/ui/CrossSell.kt

```diff
@@ -111,22 +117,37 @@ fun CrossSell(
                     )
                 }
                 Spacer(modifier = Modifier.width(8.dp))
-                Button(
-                    onClick = onCtaClick,
-                    colors = ButtonDefaults.buttonColors(
-                        backgroundColor = whiteHighEmphasis,
-                        contentColor = hedvigBlack,
-                    ),
+                CompositionLocalProvider(
+                    LocalRippleTheme provides DarkRippleTheme,
                 ) {
-                    Text(
-                        text = data.callToAction,
-                    )
+                    Button(
+                        onClick = onCtaClick,
+                        colors = ButtonDefaults.buttonColors(
+                            backgroundColor = whiteHighEmphasis,
+                            contentColor = hedvigBlack,
+                        ),
+                    ) {
+                        Text(
+                            text = data.callToAction,
+                        )
+                    }
                 }
             }
         }
     }
 }
 
+private object DarkRippleTheme : RippleTheme {
+    @Composable
+    override fun defaultColor() = Color(0x1f000000)
```

Comment:
Maybe this also deserves a comment with a link to the exact package and line of where this is originally defined in the deep theme hierarchy? Because otherwise it also feels like a magic number.

Was this the link? https://cs.android.com/android/platform/superproject/+/master:prebuilts/sdk/current/support/v7/appcompat/res/values/values.xml;l=59?q=%20lang:xml%201f000000